activerecord/lib/active_record/errors.rb

```diff
@@ -1,6 +1,47 @@
 module ActiveRecord
-  # = Active Record Errors
+  # = Active \Record \Errors
   #
+  # Subclasses <tt>ActiveModel::Errors</tt> class
+  # to implement copying association errors
+  # to the foreign key
+  class Errors < ActiveModel::Errors
+    # When passed a symbol or a name of a method, returns an array of errors
+    # for the method.
+    #
+    #   person.errors[:name]  # => ["cannot be nil"]
+    #   person.errors['name'] # => ["cannot be nil"]
+    #
+    # Note that, if you try to get errors of an attribute which has
+    # no errors associated with it, this method will instantiate
+    # an empty error list for it and +keys+ will return an array
+    # of error keys which includes this attribute.
+    #
+    #   person.errors.keys    # => []
+    #   person.errors[:name]  # => []
+    #   person.errors.keys    # => [:name]
```

Comment:
I'm not sure this documentation adds anything. Given the above comment, the description of the class should be sufficient so you can remove the documentation and add a `:nodoc:` here.